.github/workflows/flink-integration-test.yml

```diff
@@ -28,77 +27,45 @@ jobs:
               - catalogs/**
               - clients/client-java/**
               - clients/client-java-runtime/**
-              - clients/filesystem-hadoop3/**
-              - clients/filesystem-hadoop3-runtime/**
               - common/**
               - conf/**
               - core/**
               - dev/**
+              - flink-connector/**
               - gradle/**
+              - iceberg/**
```

Comment:
Flink doesn't rely on `iceberg` currently.

Reply:
Would you recommend adding this line of code after Flink supports Iceberg? I'm worried that I'll forget about it then, as it seems easy to miss

Reply:
Maybe this can be solved through comments, let me try

Reply:
Iceberg support for Flink is not on the roadmap, I prefer to remove it

Reply:
removed